### PR TITLE
OpenAir v2.0.2 release notes

### PR DESCRIPTION
**Pull request by martinpaoloni**

### CODE REVIEW

**Comment by martinpaoloni** [comment]
Do not merge until released.

File: modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc [hunk 1]
```diff
@@ -9,6 +9,26 @@ The Anypoint Connector for NetSuite OpenAir is a closed source Enterprise connec
...
+
+[%header%autowidth.spread]
+|===
+|Software | Version
+|Mule Runtime | 4.1.0 and later
+|===
+
+=== Fixed in this Release
+
+* Now working with Java 9 through Java 12, when using Mule Runtime 4.2.0.
```
Comment (rddheath) [inline modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc]: Maybe something like this: "If you are using Mule Runtime 4.2.0, you can now also use Java 9, 10, 11, or 12.

File: modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc [hunk 2]
```diff
@@ -9,6 +9,26 @@ The Anypoint Connector for NetSuite OpenAir is a closed source Enterprise connec
...
+
+=== Known Issues
+
+* Do not use Metadata as a guide in Flow Designer as it displays incomplete structures.
```
Comment (rddheath) [inline modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc]: What does "it" refer to? "Metadata" or "Flow Designer"?
Will users understand what it means to "use Metadata as a guide in Flow Designer"?
What types of structures are displayed as incomplete?
What is "Metadata" here? Is that a name or should it not be capitalized?

File: modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc [hunk 3]
```diff
@@ -9,6 +9,26 @@ The Anypoint Connector for NetSuite OpenAir is a closed source Enterprise connec
 
 xref:connectors::netsuite/netsuite-openair-connector.adoc[NetSuite OpenAir Connector User Guide]
 
+== 2.0.2 
+
+*April 26, 2019*
+
+=== Compatibility
+
+[%header%autowidth.spread]
```
Comment (rddheath) [inline modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc]: If a table contains only one row of information, then you shouldn't use a table to display that info. How about using a sentence?

File: modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc (hunk 1 above)
Comment (rddheath) [inline modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc]: Is the support for these Java versions really a fix? Or is it an upgrade to the Java support in this connector?

Comment (rddheath) [inline modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc]: This note says it's for 4.2.0. Yet, the Compatibility section states that the release notes are "compatible" with 4.1.0 and later. So, this note contradicts that, as it's not "compatible" with 4.1.0.

File: modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc [hunk 4]
```diff
@@ -9,6 +9,26 @@ The Anypoint Connector for NetSuite OpenAir is a closed source Enterprise connec
 
 xref:connectors::netsuite/netsuite-openair-connector.adoc[NetSuite OpenAir Connector User Guide]
 
+== 2.0.2 
+
+*April 26, 2019*
+
+=== Compatibility
```
Comment (rddheath) [inline modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc]: Compatibility of what with what?

File: modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc [hunk 5]
```diff
@@ -9,6 +9,26 @@ The Anypoint Connector for NetSuite OpenAir is a closed source Enterprise connec
...
+|===
+|Software | Version
+|Mule Runtime | 4.1.0 and later
+|===
+
+=== Fixed in this Release
+
+* Now working with Java 9 through Java 12, when using Mule Runtime 4.2.0.
+
+=== Known Issues
```
Comment (rddheath) [inline modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc]: There is only one known issue. So, change "Issues" to be singular.

File: modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc (hunk 1 above)
Comment (rddheath) [reply modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc]: Or am I misunderstanding what "compatibility" means in the top section of these release notes?

Comment (mcherem) [reply modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc]: Thanks for the format recommendation. Will change per rddheath.

File: modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc (hunk 2 above)
Comment (mcherem) [reply modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc]: rddheath, all good questions. I will need to follow up here.

File: modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc (hunk 4 above)
Comment (mcherem) [reply modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc]: Ooops! This is missing. I need to confirm the correct number with Dev.

File: modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc (hunk 3 above)
Comment (mcherem) [reply modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc]: That's a good question. We added this section as part of the connector template. The idea is that the user could go to the same place to find compatibility info every time. Not sure we discussed the condition where there is only a single item. I need to take this back to the connector team. I may not have an answer soon enough for this.

File: modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc (hunk 1 above)
Comment (mcherem) [reply modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc]: I thought I understood it as a fix, but need to confirm.

Comment (mcherem) [reply modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc]: No, you are understanding correctly, it's the note that's convoluted.

Comment (mcherem) [reply modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc]: I just removed it from the Header and now it is less confusing.

Comment (mcherem) [reply modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc]: They match now.

File: modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc (hunk 5 above)
Comment (mcherem) [reply modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc]: Fixed.

File: modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc (hunk 4 above)
Comment (mcherem) [reply modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc]: Fixed.

File: modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc (hunk 2 above)
Comment (mcherem) [reply modules/ROOT/pages/connector/netsuite-openair-connector-release-notes-mule-4.adoc]: Rewrite to: 
Do not use Metadata as a guide in Flow Designer as not all fields show up.

**Pull request merged**